src/test/resources/velocity/velocity-test-engine-fail.properties

```diff
@@ -0,0 +1,3 @@
+runtime.references.strict: true
+file.resource.loader.path: target/test/VelocityTemplateResolverSpec/template/
+template.file.name: LicenseFailTest.html
```

Comment:
Great suggestion! In commit 908b499838003c87111d09d88cf5d4f0fdb0b43d I moved the loading of the file to outside the `VelocityTemplateResolver` because this is not that class's concern. This also made the tests easier, so now I use plane `java.util.Properties` instances for the tests.